packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx

```diff
@@ -0,0 +1,130 @@
+            part="button"
+          >
+            <div class="va-crisis-line-inner">
+              <span class="va-crisis-line-icon" aria-hidden="true"></span>
+              <span class="va-crisis-line-text">
+                Talk to the <strong>Veterans Crisis Line</strong> now
+              </span>
+              <img
+                class="va-crisis-line-arrow"
+                src={arrowRightSvg}
+                aria-hidden="true"
+              />
+            </div>
+          </button>
+        </div>
+        <va-modal
+          modalTitle="We’re here anytime, day or night – 24/7"
+          onPrimaryButtonClick={() => this.setNotVisible()}
+          onCloseEvent={() => this.setNotVisible()}
+          visible={this.isOpen}
+          large={true}
+        >
+          <div
+            id="modal-crisisline"
+            class="va-overlay va-modal va-modal-large"
+            role="alertdialog"
+          >
+            <div class="va-crisis-panel va-modal-inner">
+              <div class="va-overlay-body va-crisis-panel-body">
+                <p>
+                  If you are a Veteran in crisis or concerned about one, connect
+                  with our caring, qualified responders for confidential help.
+                  Many of them are Veterans themselves.
+                </p>
+                <ul class="va-crisis-panel-list">
+                  <li>
+                    <i
+                      class="fa fa-phone va-crisis-panel-icon"
+                      aria-hidden="true"
+                    />
+                    {
+                      <a href="tel:988">
+                        Call <strong>988 and select 1</strong>
+                      </a>
+                    }
+                  </li>
+                  <li>
+                    <i
+                      class="fa fa-mobile-alt va-crisis-panel-icon"
+                      aria-hidden="true"
+                    />
+                    <span>
+                      Text&nbsp;
+                      <va-telephone sms contact="838255" />
+                    </span>
+                  </li>
+                  <li>
+                    <i
+                      class="fa fa-comments va-crisis-panel-icon"
+                      aria-hidden="true"
+                    />
+                    <a
+                      class="no-external-icon"
+                      href="https://www.veteranscrisisline.net/get-help-now/chat/"
+                    >
+                      Start a confidential chat
+                    </a>
+                  </li>
+                  <li>
+                    <i
+                      class="fa fa-deaf va-crisis-panel-icon"
+                      aria-hidden="true"
+                    />
+                    <p>
+                      Call TTY if you have hearing loss{' '}
+                      <strong>
+                        <va-telephone tty contact={CONTACTS.CRISIS_TTY} />
```

Comment:
Don't think we need the `tty` prop here, even though it's technically a tty number. This is what's on production:

![image](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/b089990e-c6f2-453e-b5a3-22e932a8adaf)

Reply:
That seems to me to be a bug in what is in production. We added the TTY prompt so that folks would be warned before calling a TTY number.